js/client-teambuilder.js

```diff
@@ -980,12 +980,16 @@
 				if (this.curSetList.length && !this.curSetList[this.curSetList.length - 1].species) {
 					this.curSetList.splice(this.curSetList.length - 1, 1);
 				}
+
+				var teamIsEmpty = this.curSetList.length > 0;
 				if (exports.BattleFormats) {
 					buf += '<li class="format-select">';
 					buf += '<label class="label">Format:</label><button class="select formatselect teambuilderformatselect" name="format" value="' + this.curTeam.format + '">' + (Tools.escapeFormat(this.curTeam.format) || '<em>Select a format</em>') + '</button>';
-					buf += ' <button name="validate" class="button"><i class="fa fa-check"></i> Validate</button></li>';
+					var btnClass = 'button' + (!teamIsEmpty ? ' disabled' : '');
+					var btnState = !teamIsEmpty ? ' disabled' : '';
```

Comment:
I'd say leave `btnState` alone. We can give a popup if the user actually clicks it.